vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -37,6 +37,11 @@ init_config:
   #
   # refresh_metrics_metadata_interval: 600
 
+  ## @param excluded_host_tags - list of strings - optional - default: []
```

Comment:
https://github.com/DataDog/integrations-core/pull/5201/files#diff-0316466a56da4d71e74c3f8bb7dd66deR123
We should also document the option in `instances` then.

Also, what's the reason to have this option in both `init_config` and `instances` instead of just `instances`?

Reply:
Well since we now recommend having several instances for historical and realtime metrics, I figured it'd be nice not having to define that for each instance. But at the same time wanted the ability to override it per instance.

Reply:
I see, quick note that the collection of historical metrics doesn't submit any host tags (it only collects datastore/datacenter/cluster metrics)

Reply:
I see. Well if you think it doesn't make sense to have an `init_config` for that, I can remove

Reply:
No that's ok, it doesn't hurt to keep it like this and I'm sure some users have multiple 'realtime' instances in the same config so it may have a use case.